server/routes/createGoal/createGoalController.test.ts

```diff
@@ -103,6 +106,7 @@ describe('createGoalController', () => {
       // Given
       req.params.prisonNumber = 'A1234GC'
       req.body = {}
+      req.session.addStepForms = []
```

Comment:
In this test, in the `// Then` block, we're going to want to assert the `addStepForms` array is still empty?